packages/flutter/lib/src/material/selectable_text.dart

```diff
@@ -324,6 +326,9 @@ class SelectableText extends StatefulWidget {
   /// {@macro flutter.widgets.editableText.textScaleFactor}
   final double? textScaleFactor;
```

Comment:
> I'll migrate the packages

I can't find where we document it, but we guarantee that the packages in https://github.com/flutter/packages are backwards compatible with x old stable versions. So, it will be quite some time before you can actually migrate these packages to the new API. We shouldn't wait that long to deprecate these - it will just be confusing to have to parallel APIs with no clear indicator which one to use. Typically, we just add ignores for the deprecation message to the lines they appear on in https://github.com/flutter/packages and then land the change in the framework that deprecates them.

Reply:
Created https://github.com/flutter/packages/pull/4209 to add ignores. But I'm not sure if changes like that needs a version bump or not.